target/linux/ipq40xx/patches-5.4/0020-mtd-nand-Add-support-for-Toshiba-TC58NVG0S3HTA00.patch

```diff
@@ -0,0 +1,12 @@
+--- a/drivers/mtd/nand/raw/nand_ids.c
++++ b/drivers/mtd/nand/raw/nand_ids.c
+@@ -30,6 +30,9 @@ struct nand_flash_dev nand_flash_ids[] =
+ 		{ .id = {0x98, 0xd1, 0x90, 0x15, 0x76, 0x14, 0x01, 0x00} },
+ 		  SZ_2K, SZ_128, SZ_128K, 0, 8, 64, NAND_ECC_INFO(1, SZ_512),
+ 		  2 },
++	{"TC58NVG0S3HTA00 1G 3.3V 8-bit",
++		{ .id = {0x98, 0xf1, 0x80, 0x15} },
++		  SZ_2K, SZ_128, SZ_128K, 0, 4, 128, NAND_ECC_INFO(8, SZ_512), },
```

Comment:
Please move this to target/linux/generic/ for kernel 5.10 and 5.15.
 
It would be nice if you could send this upstream too.

Reply:
Ah damn, I forgot to sync my fork! I was unsure whether it should be in platform or generic, but I'll move it to generic. I wanted to send it upstream anyways.

Reply:
Could you already send this upstream and then add into the pending folder please. Please also add the git patch header you used upstream.